src/Microsoft.SqlTools.Hosting/Hosting/Protocol/RequestContext.cs

```diff
@@ -56,7 +56,7 @@ public virtual Task SendError(string errorMessage, int errorCode = 0)
         public virtual Task SendError(Exception e)
         {
             // Overload to use the parameterized error handler
-            return SendError(e.Message, e.HResult);
+            return SendError(string.Format("{0}{1}{2}", e.Message, Environment.NewLine, e.StackTrace), e.HResult);
```

Comment:
while working on the new feature, I ran into a couple errors that only shows me the error message without stacktrace.
it would be helpful for us to debug the issue especially when users ran into issues.

Reply:
I'm not sure I like doing this for ALL errors - because we often will end up displaying these to the user. I'd suggest adding an extra StackTrace field to the Error object we're writing back instead.

Reply:
makes sense, I'll revert the change here and have a separate PR.